services/web-app/components/catalog/catalog.js

```diff
@@ -79,7 +80,7 @@ function Catalog({ collection, data, type, filter: defaultFilter = {}, glob = {}
   })
 
   const [filter, setFilter] = useQueryState('filter', {
-    ...queryTypes.object,
+    ...queryTypes.prettyObject,
     defaultValue: defaultFilter
```

Comment:
As a safe guard, we might want to clean the filter in the two places that we set it. Here and in `handleFilter()`.

```suggestion
    defaultValue: getCleanFilter(defaultFilter)
```